### DISCUSSION TITLE
Retrieve bounces from Postal

### DISCUSSION THREAD

**Discussion by Orminor77** [postalserver/postal#1843] [Help with using Postal] [answered]
Hello there,

I'm using Postal and I have a problem to recover the bounces, I would like all the bounces to be sent to an email address "bounce@domain.com" is it possible? If yes, how to do it?

Otherwise I saw that there are http endpoints but I don't understand at all how it works. Do you have any examples if the first request is impossible?

The final goal is to be able to send bounces to an "address email" or a "webhook" in order to retrieve the bounce stats on Mailwizz.

Thank you in advance for your answers and your help!

**Comment by Orminor77** [comment]
Hello again,

Nobody has an answer to this question?
I've searched and searched, but I can't find anything that matches my search, any ideas?

**Comment by willpower232** [comment, marked as answer]
Mailwizz has come up a few times in these issues so it might be worth asking them how they have dealt with it because they would be better placed to advise.

**Reply by Orminor77** [reply]
Hi, thanks you right, I will do it !
